src/pudl/transform/eiaaeo.py

```diff
@@ -626,7 +626,7 @@ class AeoCheckSpec:
 def make_check(spec: AeoCheckSpec) -> AssetChecksDefinition:
     """Turn the AeoCheckSpec into an actual Dagster asset check."""
 
-    @asset_check(asset=spec.asset)
+    @asset_check(asset=spec.asset, blocking=True)
```

Comment:
@jdangerx Is there a reason that these checks weren't marked as blocking before?